docs/reference/run.md

```diff
@@ -984,6 +984,7 @@ or override the Dockerfile's exposed defaults:
                    format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
                    Both hostPort and containerPort can be specified as a range of ports.
                    When specifying ranges for both, the number of container ports in the range must match the number of host ports in the range. (e.g., `-p 1234-1236:1234-1236/tcp`)
+                   When specifying a range for hostPort only, the containerPort must not be a range.  In this case the container port is published somewhere within the specified hostPort range. (e.g., `-p 1234-1236:1234/tcp`)
```

Comment:
@dkjer shall I suggest a minor tweak to this Doc and get it merged ?
@moxiegirl can update the doc in a later commit with the table structure that she proposed ?

The proposed documentation is confusing 

> When specifying a range for hostPort only, the containerPort must not be a range.

Confusing and it is conflicting with the other option of providing both hostPort range and containerPort range.

> In this case the container port is published somewhere within the specified hostPort range.

I would reword it as `In this case the container port is mapped to a host port somewhere within the specified hostPort range``